src/export/export_cluster_snapshot_s3_function.py

```diff
@@ -0,0 +1,52 @@
+import os
+import time
+
+import boto3
+
+import constants
+import utility as util
+
+
+def lambda_export_rds_cluster_snapshot_to_s3(event, context):
+    """start export task of RDS cluster snapshot to S3 bucket"""
+    region = os.environ['Region']
+    rds = boto3.client('rds', region)
+    result = {}
+    instance_id = event['identifier']
+    epoch = int(time.time())
+    export_id = instance_id + "-" + str(epoch)
+    snapshot_id = instance_id + constants.SNAPSHOT_POSTFIX
+    snapshot_arn = get_cluster_snapshot_arn(snapshot_id)
+    account_id = util.get_aws_account_id()
+    bucket_name = constants.RDS_SNAPSHOTS_BUCKET_NAME_PREFIX + account_id
+    try:
+        response = rds.start_export_task(
+            ExportTaskIdentifier=export_id,
+            SourceArn=snapshot_arn,
+            S3BucketName=bucket_name,
+            IamRoleArn=os.environ['SNAPSHOT_EXPORT_TASK_ROLE'],
+            KmsKeyId=os.environ['SNAPSHOT_EXPORT_TASK_KEY'],
+        )
+        result['taskname'] = constants.EXPORT_SNAPSHOT
+        result['identifier'] = instance_id
+        result['status'] = response['Status']
+        return result
+    except Exception as error:
+        raise Exception(error)
+
+
+def get_cluster_snapshot_arn(snapshot_name):
+    """returns cluster snapshot arn if in available state"""
+    region = os.environ['Region']
+    rds = boto3.client('rds', region)
+    snapshots_response = rds.describe_db_cluster_snapshots(DBClusterSnapshotIdentifier=snapshot_name)
+    assert snapshots_response['ResponseMetadata'][
+               'HTTPStatusCode'] == 200, f"Error fetching cluster snapshots: {snapshots_response}"
+    snapshots = snapshots_response['DBClusterSnapshots']
+    assert len(snapshots) == 1, f"More than one snapshot matches name {snapshot_name}"
```

Comment:
@RiyaJohn since multiple snapshots with same name isnt possible, please change the messaging to indicate that if len(snapshots) != 1, "No snapshots matches name {snapshot_name}"